mff/MFF.py

```diff
@@ -59,6 +61,27 @@ class MFF:
         Indicate whether parallelization should be employed for generating the
         first step forecasts. Default value is `True`. 
 
+    n_sample_split : int
+        Number of windows to split data into training and testing sets for 
+        generating matrix of forecast errors. Default is 5.
+
+    shrinkage_method : str, optional(default: 'oas')
+        Method to be used for shrinking sample covariance matrix. Default is 
+        Oracle Shrinking Approximating Estimator ('oas'). Other options are
+        oas, identity and monotone_diagonal.
+    
+    lamstar_empirically : boolean, optional(default: True)
+        Indicate whether the smoothness paramatere lambda is to be calculated
+        empirically 
+
+    default_lam : float, optional(default: 6.25)
```

Comment:
I think it's possible to combine `lamstar_empirically` and `default_lam`. So, e.g. if `default_lam` is 0 or -1, then the lambda is calculated empirically...